README.md

```diff
@@ -50,7 +50,16 @@ Here is how ThirdStats looks like on the Thunderbird default dark theme and ligh
 
 ## Privacy and Security
 
-ThirdStats is fully contained and doesn't contact any third-party CDN servers. It runs locally without any webserver. ThirdStats stores processed data in a cache, but you can clear and disable it. ThirdStats will never store this data elsewhere, nor sent or sell it anywhere. ThirdStats needs permissions to access accounts, read email headers and download export files. ThirdStats will always seek to minimise use of permissions where possible, and use granular permissions where available. Thunderbirds Permissions API is currently limited to access the whole email instead of the email header only.
+ThirdStats runs entirely locally and will never contact, send or sell data to, any third-parties. 
+
+ThirdStats features require [Thunderbird permissions](https://developer.thunderbird.net/add-ons/mailextensions/supported-webextension-api) to function, but will always seeks to minimise permissions where possible, and use granular permissions where available [1]:
+
+ * Accounts: Access accounts and identities (read-only)
+ * Downloads: Export and download data as a file
+ * Messages: Access messages to create statistics (read-only)
+ * Storage: Store processed data (cache) for performance. Can be cleared or disabled at any time.
```

Comment:
### Goals/Considerations:

* Prefer to be *explicit rather than implicit* (gooin **above and beyond what Thunderbird does** where possible)
* Explicit **tie and describe** 'Permission' name with **Actual and specific addon function** that needs it'. eg: `Permission: <Feature Description>`

### Questions


1) Is `Storage` *required* to enable/store the cache?
2) Can we 'opt out' of `Storage` ? If so, we'd disable the 'Cache' option with a `(i)` indicator saying why and how to enable

tldr: If storage is *required* by *any* thirdstats function/feature, *listed by thunderbird or not*, describe the permission and the feature in docs.

Reply:
Wow always a pleasure to read your comments highlighting the problem with a minimal amount of words 😍 Totally agree with your proposed goals and will merge this PR as it is 👍🏻 

1. Yes, `Storage` is required to use the extension storage (= ThirdStats cache) at all
2. We can't afaik. Permissions are given by the user on installing the add-on and cannot be revoked after.